tests/core/remodel-kernel/task.yaml

```diff
@@ -16,7 +16,9 @@ prepare: |
     . "$TESTSLIB"/systemd.sh
 
     # Save the revision of the pc-kernel snap.
-    readlink /snap/pc-kernel/current > original-revision.txt
+    readlink /snap/"$OLD_KERNEL"/current > original-revision.txt
+    # Save the original tracking channel
+    snap list "$OLD_KERNEL" | grep -v ^Name | tr -s ' ' | cut -f4 -d' ' > original-channel.txt
```

Comment:
```suggestion
    snap list "$OLD_KERNEL" | grep -v ^Name | awk '{print $4}' > original-channel.txt
```

Reply:
wouldn't snap info and grep tracking:  be slightly simpler?  not sure

Reply:
Nice suggestions, tweaked it to use both which makes it much nicer.